website/docs/r/destination_of_redirected_traffic.html.markdown

```diff
@@ -31,14 +39,14 @@ resource "aci_destination_of_redirected_traffic" "example" {
 
 ## Argument Reference
 
-- `service_redirect_policy_dn` - (Required) Distinguished name of parent Service Redirect Policy object.
+- `service_redirect_policy_dn` - (Required) Distinguished name of the parent Service Redirect Policy object.
 - `ip` - (Required) The IP address.
-- `mac` - (Required) The MAC address.
-- `annotation` - (Optional) Annotation for object destination of redirected traffic.
-- `description` - (Optional) Description for object destination of redirected traffic.
-- `dest_name` - (Optional) The destination name to which data was exported. This utility creates a summary report containing configuration information, logs and diagnostic data that will help TAC in troubleshooting and resolving a technical issue.
-- `ip2` - (Optional) IP2 for object destination of redirected traffic. Default value: "0.0.0.0"
-- `name_alias` - (Optional) Name alias for object destination of redirected traffic.
+- `mac` - (Optional) The MAC address. This is a required value for APIC prior to Version 5.2 release. This value can be foregone by enabling IPSLA on APIC Version 5.2 and above due to dynamic mac detection feature.
+- `annotation` - (Optional) Annotation for the object destination of redirected traffic.
+- `description` - (Optional) Description for the object destination of redirected traffic.
```

Comment:
for the object destination of redirected traffic -> of the destination of redirected traffic object.